CHANGELOG.md

```diff
@@ -244,6 +252,21 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 - `AHuman:ReloadFirearm` Lua function has been changed to `AHuman:ReloadFirearms` and will now reload offhand weapons as well, if appropriate.
 
+- `Settings.ini` property `MenuTransitionDuration` renamed to `MenuTransitionDurationMultiplier`.
+
+- `Settings.ini` property `DisableLoadingScreen` renamed to `DisableLoadingScreenProgressReport`.
+
+- Scenario scene markers are now color coded to help distinguish them visually:  
+	`Base.rte` scenes are yellow as always.  
+	`Missions.rte` scenes are now green.  
+	`Scenes.rte` or any other mod/user scenes are now cyan.
+
+- Main menu and sub-menus were given a facelift.
+
+- In-game settings menu was reworked to make it less useless.
```

Comment:
lol it just occurs to me that the wording of this might suggest that this settings menu is when you're actually playing a game, or that we have an out-of-game settings menu :O